model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -337,24 +337,19 @@ message Annotation {
 // MonitoringInfo protos.
 message MonitoringInfoSpecs {
   enum Enum {
-    // TODO(ajamato): Add the PTRANSFORM name as a required label after
```

Comment:
Let's keep this one in please. @Ardagan's validation is going to pull in these strings soon. We are hoping to remove the MonitoringInfoUrns section and just have MonitoringInfoSpecs

Reply:
hm, @robertwb [wanted it removed](https://github.com/apache/beam/pull/7868#discussion_r257840224), IIUC.

Reply:
The problem is that "beam:metric:user:" isn't actually a counter, nor does it have a specific type, so putting it in this list seems wrong. It's just a prefix that we assert will not be used for any runner-generated counters. 

[If we do remove it, no need to renumber everything else.]

Reply:
> [If we do remove it, no need to renumber everything else.]

`generateProto` fails without the renumbering:

```
Execution failed for task ':beam-model-fn-execution:generateProto'.
> protoc: stdout: . stderr: beam_fn_api.proto: The first enum value must be zero in proto3.
```

Reply:
(update: it seems if I only renumber the first entry to `0`, I can skip `1` and leave the others untouched. Given that these aren't really used yet, it feels cleaner to just renumber them all now)

Reply:
My take on this:
1. It is better ot keep this spec present. It defines what fields user counter needs.
2. We can rename this element to something like USER_COUNTER_PREFIX for clarity.
3. It is ok to move it away from this list, but we should still keep it somewhere for generic validation and specification. However in this case we might want to move it away from MonitoringInfoUrns as well.
4. Currently, we have urns defined in this list and in MonitoringInfoUrns. I was thinking of removing MonitoringInfoUrns completely and keep only MonitoringInfoSpecs as explicit source of truth.